planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -17,6 +17,27 @@
           "└─TableReader_6 10000.00 root  data:TableFullScan_5",
           "  └─TableFullScan_5 10000.00 cop[tiflash] table:t keep order:false, stats:pseudo"
         ]
+      },
+      {
+        "SQL": "explain select * from t where b > 'a' order by convert(b, unsigned) limit 2",
+        "Plan": [
+          "Projection_17 2.00 root  test.t.a, test.t.b",
+          "└─TopN_8 2.00 root  Column#3, offset:0, count:2",
+          "  └─Projection_18 3333.33 root  test.t.a, test.t.b, cast(test.t.b, bigint(22) UNSIGNED BINARY)->Column#3",
+          "    └─TableReader_13 3333.33 root  data:Selection_12",
+	  "      └─Selection_12 3333.33 cop[tiflash]  gt(test.t.b, \"a\")",
```

Comment:
please use spaces instead of tabs to indent the file.

Reply:
Fixed.